.circleci/report_coverage.rb

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
```

Comment:
I dont know if we need to be consistent, but in other projects where I have created executables for use in CI, I have used `bin/ci` to store the executables.

Reply:
Currently the CI executables for this project are in the `.circleci` directory. I'm fine with refactoring the directory structure for the project to follow a more generic standard, but I think that's out of the scope of this PR. Maybe make a card in Jira and we can address this issue, as well as the directory structure as a whole.

Reply:
ok. that's fine. I just wanted to bring it up before it was too late